New package: DifferentiationInterfaceTest v0.1.0

JuliaRegistrator opened the pull request:
- Registering package: DifferentiationInterfaceTest
- Repository: https://github.com/gdalle/DifferentiationInterface.jl
- Created by: @gdalle
- Version: v0.1.0
- Commit: 9791a107d41c673c31ae279176788bc0805ae20f
- Git reference: HEAD
- Description: An interface to various automatic differentiation backends in Julia.

gdalle commented:
[noblock] will be fixed by https://github.com/gdalle/DifferentiationInterface.jl/pull/162

JuliaHubRegistrator committed:
New package: DifferentiationInterfaceTest v0.1.0

UUID: a82114a7-5aa3-49a8-9643-716bb13727a3
Repo: https://github.com/gdalle/DifferentiationInterface.jl.git
Tree: e3e6be239208065527df3e59c68915a9717ee2db

Registrator tree SHA: 7242ef64be5953fac9bf8b3efa3a7c4d2d44ca09
